openmc/tallies.py

```diff
@@ -1425,25 +1426,47 @@ def export_results(self, filename='tally-results', directory='.',
             # Pickle the Tally results to a file
             pickle.dump(tally_results, open(filename, 'wb'))
 
-    def _outer_product(self, other, binary_op):
+    def hybrid_product(self, other, binary_op, filter_product='None',
```

Comment:
My intention of having the string "None" was to avoid having the input be either of type str *or* NoneType and just have it as a str. I can see the "None" string being a bit confusing given Python has a None type so I'll go ahead and change it to `None`.